docs/docs/build/examples/btc_swap/index.md

```diff
@@ -3,32 +3,143 @@
 This example demonstrates how BOB can be used to build a peer to peer swap application which allows BTC to be exchanged for an ERC20 token without the use of a custodian.
 
 :::note
+This example app is a work in progress, and the application repository is currently set to private.
+:::
+
+## Demo
+
+We have created a testnet demo of the Peer to Peer swap which allows you to swap BTC for ERC20 tokens, ERC20 tokens for BTC, and exchange one ERC20 token for another. The application currently supports USDT and ZBTC.
+
+### Connecting your wallet
+
+- Go to [demo.gobob.xyz](https://demo.gobob.xyz)
+- Connect to your wallet using the 'Connect Wallet' buttton at the top right of the UI.
+
+#### Known issue
+
+The UI can sometimes fail to connect if you have multiple wallets installed. If this happens you should disable all wallet extensions other than the one you want to use for the demo.
+
+![connect](connect-wallet.png)
+
+### Creating and funding test accounts
+
+:::note
+ZBTC tokens will be imported into your wallet with a ticker of bBTC. These will show as ZBTC tokens in the application.
+:::
```

Comment:
```suggestion

:::
```

Reply:
Fixed this manually so that I could fix it for all notes at once